.changeset/fix-inputWrapper.md

```diff
@@ -0,0 +1,5 @@
+---
+'react-magma-dom': patch
+---
+
+fix(inputbase): Add and update width prop for InputBase
```

Comment:
InputBase is used under the hood for many components, but adopters don't know this. We should mention the specific components it impacts: Input, PasswordInput.

Does this need to also be applied to Search?